src/components/config/HostInput.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from "react";
+import styled from "styled-components";
+import { Perlin } from "../../Perlin";
+import "./config.scss";
+
+const Input = styled.input`
+    outline: none;
```

Comment:
missing some semi-colons here. I recommend using the `vscode-styled-components` extension if you're using vscode.